docs/overview/sapi.rst

```diff
@@ -89,22 +258,29 @@ runs these same configuration steps):
   :start-after: cli-example-config-start-marker
   :end-before: cli-example-config-end-marker
 
-2. Enter the SAPI information (e.g. your API token) found as described in the section
-   above. To get started, create a minimum configuration by accepting the command's
-   defaults (pressing Enter) for all prompts except the API token (Leap users) or
-   API token and endpoint (on-premises users). You can in the future update the
-   file if needed.
+2. Enter the SAPI information (e.g. your API token) found as described in the
+   :ref:`sapi_access_interacting` section above. To get started, create a minimum
+   configuration by accepting the command's defaults (pressing Enter) for all
+   prompts except the API token\ [#]_\ . You can in the future update the file
+   if needed.
+
+.. [#] Users of on-premises systems should also enter the SAPI endpoint. Users
+   interested in using solvers hosted outside their default region can also
+   configure the SAPI endpoint (see the :ref:`sapi_intro_multiregion` section for
+   the recommended way of configuring such access).
 
 Alternatively, you can create and edit a
 :doc:`D-Wave Cloud Client configuration file </docs_cloud/sdk_index>`
```

Comment:
For both of these I'm linking to cloud-client generically because many of this content's users will be first-time installers so probably best not to send them deep into cloud-client. That is why most the content uses dwave CLI and this alternative is just mentioned for completeness.

They see the `dwave config ls` a few lines above:
![image](https://user-images.githubusercontent.com/34041130/140830201-7c912ed0-02f0-4b6b-a66c-4ab0bba7b769.png)